packages/overlays/src/OverlayMixin.js

```diff
@@ -150,23 +169,41 @@ export const OverlayMixin = dedupeMixin(
       __setupSyncFromOverlayController() {
         this.__onOverlayCtrlShow = () => {
           this.opened = true;
+          this.dispatchEvent(new Event('opened'));
```

Comment:
maybe move it to `_requestUpdate`?
ala
```js
_requestUpdate(name, oldValue) {
  super._requestUpdate(name, oldValue);
  if (name === 'opened) {
    this.dispatchEvent(new Event('opened-changed'));
  }
}
```

OR
use `this.dispatchEvent(new Event('opened-changed'));` in `__onOverlayCtrlShow ` and `__onOverlayCtrlHide`